erpnext/setup/doctype/currency_exchange_settings/test_currency_exchange_settings.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017, Frappe Technologies Pvt. Ltd. and Contributors
+# See license.txt
+from __future__ import unicode_literals
+
+import frappe
+import unittest
+
+
+class TestCurrencyExchangeSettings(unittest.TestCase):
+	def test_change_setting(self):
+		settings_doc = frappe.get_doc("Currency Exchange Settings", "Currency Exchange Settings")
```

Comment:
@tundebabzy such tests are not needed. You should only write tests to test code in your own controller.

Also this can also be part of "Accounts Settings"